docs/intro/migrations.rst

```diff
@@ -21,69 +39,77 @@ your codebase.
 
   .
   ├── dbschema
-  │   ├── default.esdl          # schema file (written by you)
-  │   └── migrations            # migration files (typically generated by CLI)
-  │       └── 00001.edgeql
-  │       └── ...
+  │   └── default.esdl          # schema file (written by you)
   └── edgedb.toml
 
-The schema itself is written using EdgeDB's schema definition language.
+The schema itself is written using EdgeDB's schema definition language. Edit
+your ``dbschema/default.esdl`` and add the following schema inside your
+``module default`` block:
 
 .. code-block:: sdl
 
   type User {
-    required property name -> str;
+    required name: str;
   }
 
   type Post {
-    required property title -> str;
-    required link author -> User;
+    required title: str;
+    required author: User;
   }
 
 
-It's common to keep your entire schema in a single file, typically called
-``default.esdl``. However it's also possible to split it across a number of
-``.esdl`` files.
-
-To spin up a new instance and populate it with an initial schema, execute the
-following commands in a fresh directory.
+It's common to keep your entire schema in a single file, and many users use
+this ``default.esdl`` that is created when you start a project. However it's
+also possible to split their schemas across a number of ``.esdl`` files.
 
-.. code-block:: bash
+Once you save your initial schema, assuming it is valid, the ``watch`` command
+will pick it up and apply it to your database.
 
-  $ edgedb project init
-  Do you want to initialize a new project? [Y/n]
-  > Y
-  <additional prompts>
-  $ edgedb migration create
-  Created dbschema/migrations/00001.edgeql (id: <hash>)
-  $ edgedb migrate
-  Applied dbschema/migrations/00001.edgeql (id: <hash>)
 
 2. Edit your schema files
 -------------------------
 
 As your application evolves, directly edit your schema files to reflect your
-desired data model.
+desired data model. Try updating your ``dbschema/default.esdl`` to add a
+``Comment`` type:
 
 .. code-block:: sdl-diff
 
     type User {
-      required property name -> str;
+      required name: str;
     }
 
-    type BlogPost {
-      property title -> str;
-      required link author -> User;
+    type Post {
+      required title: str;
+      required author: User;
     }
 
   + type Comment {
-  +   required property content -> str;
+  +   required content: str;
   + }
 
+When you save your changes, the updates will be immediately applied to your
```

Comment:
Is it okay to say "immediately" when in reality it's 3-5-10 seconds?

Reply:
The etymologist in me likes immediately since at its core it means "without mediation" / "nothing in between" but I guess that meaning is pretty rare now. Maybe a move to active tense and change to something like "immediately move to apply" could be good:

When you save your changes, ```watch``` will immediately move to apply the changes to your database as long as it is still running.

Reply:
I like the suggestion, @Dhghomon. What about "When you save your changes, the CLI will immediately begin applying the changes to your..."?

Reply:
That looks good too, yeah. The only change I might suggest then is to mention the CLI once a bit earlier so there's no mistaking that `watch` is happening through the CLI. Perhaps here:

This long-running task will monitor your schema files

change to

This long-running task uses the CLI to monitor your schema files

or

This long-running task gets the CLI to monitor your schema files

Or something along those lines.